clients/da-vinci-client/src/main/java/com/linkedin/davinci/storage/chunking/ChunkedValueManifestContainer.java

```diff
@@ -0,0 +1,19 @@
+package com.linkedin.davinci.storage.chunking;
+
+import com.linkedin.venice.storage.protocol.ChunkedValueManifest;
+
+
+public class ChunkedValueManifestContainer {
```

Comment:
This can be a generic container to keep any type of object, right?
Such as:
```
public class ObjectContainer<T>
```

Reply:
Hmm...why? Here we are always targeting Manifest type right? No matter it is value or RMD

Reply:
I was trying to see whether we can create this class as a utility class, which can be used by other places.

Reply:
I think let's leave it as it is for now. We could potentially refactor it if we do see more general use case there...